configs/processor/flow_matching.yaml

```
@@ -11,8 +11,11 @@ backbone:
   in_channels: null
   out_channels: null
   cond_channels: null
+  n_steps_output: null
+  n_steps_input: null
   mod_features: 256
   hid_channels: [32, 64, 128]
   hid_blocks: [2, 2, 2]
   spatial: 2
   periodic: false
+  temporal_method: "attention"
```

Related to comment on default below
```suggestion
  temporal_method: "none"
```